src/lib/rest/rest.cpp

```diff
@@ -370,12 +370,12 @@ static Format wantedOutputSupported(const std::string& apiVersion, const std::st
 
      std::string format = vec[ix].c_str();
      if (format == "*/*")              { xml  = true; json = true; text=true;}
-     if (format == "*/xml")            xml  = true;
+     if (format == "*/xml")            { xml  = true; }
      if (format == "application/*")    { xml  = true; json = true; }
-     if (format == "application/xml")  xml  = true;
-     if (format == "application/json") json = true;
-     if (format == "*/json")           json = true;
-     if (format == "text/plain")       text = true;
+     if (format == "application/xml")  { xml  = true; }
```

Comment:
I guess that we can break a little bit the rule that mandates to have `{` and `}` is separated lines ;)